src/util.cpp

```diff
@@ -8,6 +8,11 @@
  * @copyright Copyright (c) 2023
  * 
  */
+
+/*! \def BYTES_PER_LINE
+    \brief A macro that stores the the size of each line of output as a constant 16 bytes
+*/
+
```

Comment:
This seems to be in the wrong place...  This should be directly in front of the member you are trying to document.